### DISCUSSION TITLE
how to add local model instead of the official ones

### DISCUSSION THREAD

**Discussion by abdelkareemkobo** [openai/whisper#1464] [Q&A] [answered]
I am trying to use whisper to transcribe my data I want to use the following code 
```
from transformers import AutoProcessor, AutoModelForSpeechSeq2Seq

processor = AutoProcessor.from_pretrained("arbml/whisper-largev2-ar")

model = AutoModelForSpeechSeq2Seq.from_pretrained("arbml/whisper-largev2-ar")

result = model.transcribe("/kaggle/input/testing-audio/oh.wav")
print(result["text"])
```
It gives me "AttributeError: 'WhisperForConditionalGeneration' object has no attribute 'transcribe'"
I wonder if i can use this model with the following code from the current repo 
```
import whisper

model = whisper.load_model("base")
result = model.transcribe("audio.mp3")
print(result["text"])
```
The model is in pytorch form

**Comment by phineas-pta** [comment]
your title is a bit misleading, u r using `transformers` interface

see my comment for example usage

https://github.com/openai/whisper/discussions/654#discussioncomment-5920652

**Reply by abdelkareemkobo** [reply]
I don't want ot use transformers interface I want to use the model in the their hub i donwloaded it and tryied to add to the official whisper repo to use the CLI and other stuff from it but can't do this. 
 I tried your code it's very helfpul and this really what i need! Thanks for it but i get an error while using the fine-tuned model from huggingface with 
──────────────────
ValueError: You are trying to return timestamps, but the generation config is not properly set.Make sure to 
initialize the generation config with the correct attributes that are needed such as `no_timestamps_token_id`.For 
more details on how to generate the approtiate config, refer to 
https://github.com/huggingface/transformers/issues/21878#issuecomment-1451902363
──────────────────
```
from transformers import pipeline
import torch
device = "cuda" if torch.cuda.is_available() else "cpu"

model = pipeline(
    task="automatic-speech-recognition",
    model="arbml/whisper-medium-ar",
    device=device,
    chunk_length_s=30,
    generate_kwargs = {"num_beams":5}#same as setting as "open whisper" default
)
result = model("/content/001.mp3",return_timestamps=True,generate_kwargs={"language":"arabic","task":"transcribe"})
````
can you help me how to resolve this error?

**Reply by phineas-pta** [reply, marked as answer]
seem like your model isn't configured to return timestamps

you can simply remove `return_timestamps=True`

if you want to use whisper CLI, you have to edit source code to load fine-tuned checkpoint, you can inspire from this https://github.com/openai/whisper/discussions/830#discussioncomment-4652413
